### DISCUSSION TITLE
reused word hint ?

### DISCUSSION THREAD

**Discussion by gregy4** [google/or-tools#3472] [CP-SAT questions] [answered]
Hello,
  in a log I found ...
       #164    61.01s best:6673  next:[6544,6672] scheduling_resource_windows_lns_default(d=0.37 s=169 t=0.10 p=0.50) [hint]

  It seems that for 164. solution was used some kind of hint but my model doesn't contain hints. When I used them, [hint] was displayed for first solution only ...

  I use stable version of ortools 9.4.

Jan

**Comment by lperron** [comment, marked as answer]
In LNS, we hint the previous solution when optimizing a fragment.
